command/operator_usage.go

```diff
@@ -242,6 +242,9 @@ func (c *OperatorUsageCommand) parseNamespaceCount(rawVal interface{}) (UsageRes
 		return ret, errors.New("missing non_entity_tokens")
 	}
 
+	// don't error if secret syncs are 0
+	ret.secretSyncs, _ = jsonNumberOK(counts, "secret_syncs")
```

Comment:
Is there a possibility to have 0 distinct entities too?

Reply:
I should make this comment clearer. The problem isn't that there are 0, it's that `secret_syncs` isn't a key in the output. This could happen if your Vault CLI version is newer than your Vault server version.